.teamcity/_self/projects/WebApp.kt

```diff
@@ -797,8 +797,8 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
 
 				# Run the test
-				export BROWSERSIZE="mobile"
-				export BROWSERLOCALE="en"
+				export DISPLAYSIZE="mobile"
+				export TARGETLOCALE="en"
```

Comment:
Same concern here about the `target` keyword - do you think it would be OK to drop it? i.e.

- `BROWSERLOCALE` -> `LOCALE`
- `getTargetLocale` -> `getLocale`

Reply:
Agreed.